src/system/NewTabs/Tabs.js

```diff
@@ -0,0 +1,28 @@
+/** @jsxImportSource theme-ui */
+
+/**
+ * External dependencies
+ */
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import * as TabsPrimitive from '@radix-ui/react-tabs';
+
+/**
+ * Internal dependencies
+ */
+
+const NewTabs = ( { defaultValue = null, className = null, sx, ...props } ) => (
```

Comment:
Currently, we are avoiding having `...props`, since this creates confusion about the components' API. If we don't need all radix API, we can use only what is needed?